### PR TITLE
Add expandable container

### PR DESCRIPTION
**Pull request by AdityaKhatri**

### CODE REVIEW

**Review by samshara — APPROVED** [review]

File: src/components/ExpandableContainer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    } = props;
+
+    const [
+        showContent,,,,
+        toggleContentVisibility,
+    ] = useBooleanState(defaultVisibility);
+
+    const mountContent = useMemo(() => (
+        mount ? true : showContent
+    ), [mount, showContent]);
```
Comment (tnagorra) [inline src/components/ExpandableContainer/index.tsx]: Why memoize this?

Comment (tnagorra) [reply src/components/ExpandableContainer/index.tsx]: Why not just use `||`?

File: src/components/ExpandableContainer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    className?: string;
+    childrenContainerClassName?: string;
+    headerContainerClassName?: string;
+    headingClassName?: string;
+    descriptionClassName?: string;
+    description?: React.ReactNode;
+    heading?: React.ReactNode;
+    children: React.ReactNode;
+    defaultVisibility?: boolean;
+    mount?: boolean;
```
Comment (tnagorra) [inline src/components/ExpandableContainer/index.tsx:24]: What does mount do?

**Review by tnagorra — CHANGES_REQUESTED** [review]
Please document usage for mount

File: src/components/ExpandableContainer/index.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    className?: string;
+    childrenContainerClassName?: string;
+    headerContainerClassName?: string;
+    headingClassName?: string;
+    descriptionClassName?: string;
+    description?: React.ReactNode;
+    heading?: React.ReactNode;
+    children: React.ReactNode;
+    defaultVisibility?: boolean;
+    // NOTE: Mount will mount the child even if its not shown
```
Comment (tnagorra) [inline src/components/ExpandableContainer/index.tsx:23]:
```suggestion
    // NOTE: Mount will mount the child even if its not visible
```

**Pull request merged**